CHANGELOG/CHANGELOG-1.7.md

```diff
@@ -1,3 +1,11 @@
+# v1.7.6 - Changelog since v.1.7.5
```

Comment:
can you rebase to master to make sure that the 1.7.6 change don't get messed up or double applied or something? I thought the diff will automatically get rid of this part since #1198 has merged

Reply:
Done